library/amqp.cc

```diff
@@ -115,7 +112,7 @@ namespace dripline
     {
         if( a_value.is_bool() ) return AmqpClient::TableValue( a_value.as_bool() );
         if( a_value.is_int() ) return AmqpClient::TableValue( a_value.as_int() );
-        if( a_value.is_uint() ) return AmqpClient::TableValue( a_value.as_uint() );
+        if( a_value.is_uint() ) return AmqpClient::TableValue( (uint32_t)a_value.as_uint() );
```

Comment:
Don't the other changes convert all the `as_uint()` to do a 32 already? Or is this a scarab vs amqp thing?

Reply:
This is an AMQP doesn't deal with 64-bit unsigned ints thing.